common/test/acceptance/tests/lms/test_lms_dashboard.py

```diff
@@ -153,7 +153,8 @@ def test_dashboard_social_sharing_feature(self):
         Validate the behavior of the social sharing feature
         """
         twitter_widget = self.dashboard_page.get_course_social_sharing_widget('twitter')
-        twitter_url = "https://twitter.com/intent/tweet?text=Testing+feature%3A%20http%3A%2F%2Fcustom%2Fcourse%2Furl"
+        twitter_url = ("https://twitter.com/intent/tweet?text=Testing+feature%3A%20http%3A%2F%2Fcustom%2Fcourse%2Furl"
+                       "%3Futm_campaign%3Dsocial-sharing%26utm_medium%3Dsocial-post%26utm_source%3Dtwitter")
```

Comment:
This URL doesn't look right.  Why are there 2 question marks (`%3F` and `?`) in the URL?
Also, why are you URL encoding here the `&`?